Javascript.js

```diff
@@ -1,49 +1,59 @@
-var http = require('http'),
-    path = require('path'),
-    methods = require('methods'),
-    express = require('express'),
-    bodyParser = require('body-parser'),
-    session = require('express-session'),
-    cors = require('cors'),
-    passport = require('passport'),
-    errorhandler = require('errorhandler'),
-    mongoose = require('mongoose');
-
-var isProduction = process.env.NODE_ENV === 'production';
+import express from 'express';
+import session from 'session';
+import cors from 'cors';
+import errorhandler from 'errorhandler';
+import mongoose from 'mongoose';
+import morgan from 'morgan';
+import methodOverride from 'method-override';
+import path from 'path';
+
+import './models/User';
+import './models/Article';
+import './models/Comment';
+import './config/passport';
+
+
+const isProduction = process.env.NODE_ENV === 'production';
 
 // Create global app object
-var app = express();
+const app = express();
 
 app.use(cors());
 
 // Normal express config defaults
-app.use(require('morgan')('dev'));
+app.use(morgan('dev'));
+
+app.use(methodOverride());
 
-app.use(require('method-override')());
-app.use(express.static(__dirname + '/public'));
+// Use path.join to join segments
+const directory = path.join(__dirname, '/public');
+app.use(express.static(directory));
 
-app.use(session({ secret: 'conduit', cookie: { maxAge: 60000 }, resave: false, saveUninitialized: false  }));
+app.use(
+  session({ 
+    secret: 'conduit', 
+    cookie: { maxAge: 60000 }, 
+    resave: false, 
+    saveUninitialized: false  
+  })
+);
 
 if (!isProduction) {
   app.use(errorhandler());
```

Comment:
This statement could have been in the else statement of the following if rather than having two if condition